microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy.yml

```diff
@@ -135,7 +135,7 @@ methods:
       content: 'setLocalProperty(propertyName: string, propertyValue: any): any;'
       parameters:
         - id: propertyName
-          description: Name of the property to get the value of.
+          description: Name of the property of which to set the value.
           type: string
         - id: propertyValue
           description: Value to set the property to.

```

Comment:
line 146: The link "Extends CallableFunction" is 404.
That link in live docs is in:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy

Reply:
Closing comment.  Dave is researching where CallableFunciton is defined.